languages/elixir/exercises/concept/rpn-calculator-output/test/rpn_calculator/output_test.exs

```diff
@@ -35,16 +34,24 @@ defmodule RPNCalculator.OutputTest do
     E.g.) resource.open(filename)
     """
     test "opens resource" do
-      RPNCalculator.Output.write(@resource, @filename, @equation)
-      assert_received {:open, @filename}, @use_open_error_message
+      resource = __MODULE__
+      filename = "filename"
+      equation = "1 1 +"
+
+      RPNCalculator.Output.write(resource, filename, equation)
+      assert_received {:open, ^filename}, @use_open_error_message
     end
 
     @tag :pending
     @use_write_error_message """
     Use IO.write/2 to write to the opened `filename`.
     """
     test "writes to resource" do
-      assert capture_io(fn -> RPNCalculator.Output.write(@resource, @filename, @equation) end) ==
+      resource = __MODULE__
+      filename = "filename"
+      equation = "1 1 +"
+
+      assert capture_io(fn -> RPNCalculator.Output.write(resource, filename, equation) end) ==
                "1 1 +",
              @use_write_error_message
```

Comment:
IMO this can stay in a module attribute because the string will be printed in the output in case of a failing test anyway.